src/doc/rustdoc/src/linking-to-items-by-name.md

```diff
@@ -19,15 +20,24 @@ pub struct Foo3;
 /// This struct is also not [`Bar`]
 pub struct Foo4;
 
+/// This struct *is* [`Bar`]!
 pub struct Bar;
 ```
 
-You can refer to anything in scope, and use paths, including `Self`, `self`, `super`, and `crate`. You may also use `foo()` and `foo!()` to refer to methods/functions and macros respectively. Backticks around the link will be stripped.
+Backticks around the link will be stripped, so ``[`Option`]`` will correctly
+link to `Option`.
+
+You can refer to anything in scope, and use paths, including `Self`, `self`,
+`super`, and `crate`. You may also use `foo()` and `foo!()` to refer to methods/functions and macros, respectively.
+
+You can also refer to items with generic parameters like `Vec<T>`. The link will
+resolve as if you had written ``[`Vec<T>`](Vec)``. Fully-qualified syntax (for
+example, `<Vec as IntoIterator>::into_iter()`) is not yet supported, however.
```

Comment:
Can you add a link to https://github.com/rust-lang/rust/issues/74563 ? Also I'm not sure what the policy is on documenting things that _aren't_ supported.

Reply:
We could remove that sentence since using FQS will cause an error, but this way people don't get surprised by it.